grib/src/main/java/ucar/nc2/grib/collection/GribTimeCoordIntervalUtils.java

```diff
@@ -0,0 +1,75 @@
+package ucar.nc2.grib.collection;
+
+import ucar.nc2.grib.coord.TimeCoordIntvValue;
+
+import java.util.List;
+
+public class GribTimeCoordIntervalUtils {
```

Comment:
Since this is only being used in this package, we could make this package private
```suggestion
class GribTimeCoordIntervalUtils {
```

Reply:
Good idea.